frontend-react/src/components/DataDashboard/FacilitiesProviders/FacilitiesProviders.module.scss

```diff
@@ -8,19 +8,23 @@
     }
 
     hr {
-        border: 1px solid #73B3E7;
+        border: 1px solid #73b3e7;
     }
 
     :global {
         .usa-breadcrumb {
-            background-color: #D9E8F6;
+            background-color: #d9e8f6;
```

Comment:
can we use the color token? `blue-10`

Reply:
I tried to and it doesn't get read from this file.